packages/office-ui-fabric-react/src/components/__snapshots__/Rating.Basic.Example.tsx.shot

```diff
@@ -28,6 +28,12 @@ exports[`Component Examples renders Rating.Basic.Example.tsx correctly 1`] = `
           height: 36px;
         }
     id="Rating0"
+    max={5}
+    min={1}
+    onBlur={[Function]}
+    onChange={[Function]}
+    onFocus={[Function]}
+    size={1}
```

Comment:
or size for that matter.

Reply:
I'm not sure how size got there, but i see the problem with min and max, and have a local fix. The properties list hasn't been touched in 3 years, but there are a lot of things in there that shouldn't be. I'll fix that separately.

Reply:
@dzearing Do I need to make any changes to PR as a result of yours?

Reply:
@marygans I just merged. So pull master into your branch. Then re-build, then update snapshots again. You should see less weird props.

Reply:
:clap: